dist/validation/date-format-validation-error.d.ts

```diff
@@ -1,4 +1,4 @@
 export default class DateFormatValidationError extends ElementValidationError {
-    get message(): string;
+    get message(): any;
```

Comment:
Why does this need to be changed to `any`?
Ideally, we keep it as specific of a type as possible. Having to code around `any` feels like a lot of work and potentially increases the chance of an error.
Same with the other messages functions below.

Reply:
That's in a dist file . . . I don't know how or why that changed. But I made a _lot_ of little incremental changes. Let me see if I can clean up the changes to the dist files.

Reply:
I can manually change that file, but the next person who does a `yarn build` is going to stomp on it.

Reply:
Oh strange it's changed in a dist file. I wonder why the transpiler things it should be any now. 😟